lib/validates_zipcode/cldr_regex_collection.rb

```diff
@@ -162,7 +162,107 @@ module CldrRegexpCollection
       TC: /\ATKCA 1ZZ\z/,
       WF: /\A986\d{2}\z/,
       XK: /\A\d{5}\z/,
-      YT: /\A976\d{2}\z/
+      YT: /\A976\d{2}\z/,
+      AE: /\A([a-zA-Z\d\s]){3,}\z/,
```

Comment:
Actually in one of my project, I used this gem and got some more requirements for other countries too. So I manually created the regex based on its respective format. As per my knowledge its correct format and we can go ahead and add.